scripts/tools/fetch-pull-request

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+if [ "$1" == "--help" ]; then
+    printf "Usage: fetch-pull-request [--help] pull_request_id [remote=upstream] \n\n Locally fetch a pull request and test it. \n Remote is set to upstream by default.\n"
+    exit 1
+elif [ $# -eq 0 ]; then
+    echo "Pull request id not given."
+    exit 1
+else
+    set -e -x
```

Comment:
i thought set -x -e would be better if we add it only after the main part of code is being implemented or it shows if statements,echo and exit also ,should i change it ?